### PR TITLE
feat: add pangoro chain

### PR DESCRIPTION
**Pull request by sxlwar**
bump Darwinia types to 2.6.8-5

### CODE REVIEW

File: packages/apps-config/src/api/spec/index.ts [hunk 1]
```diff
@@ -9,5 +9,6 @@ import pangolin from './pangolin';
 export default {
   Crab: crab,
   Darwinia: darwinia,
-  Pangolin: pangolin
+  Pangolin: pangolin,
+  Pangoro: pangolin
```
Comment (hackfisher) [inline packages/apps-config/src/api/spec/index.ts]:
```suggestion
  Pangoro: pangoro
```

File: packages/apps-config/src/ui/general/index.ts [hunk 2]
```diff
@@ -55,7 +55,8 @@ const logoBgColors: Record<string, any> = [
   ['Darwinia CC2', chainLogoDarwinia],
   ['Darwinia CC3', chainLogoDarwinia],
   ['Darwinia Devnet', chainLogoDarwinia],
-  ['Pangolin', chainLogoPangolin]
+  ['Pangolin', chainLogoPangolin],
+  ['Pangoro', chainLogoPangolin]
```
Comment (hackfisher) [inline packages/apps-config/src/ui/general/index.ts]:
```suggestion
  ['Pangoro', chainLogoPangoro]
```

File: packages/apps-config/src/api/spec/index.ts (hunk 1 above)
Comment (sxlwar) [reply packages/apps-config/src/api/spec/index.ts]: Pangoro use the same spec with pangolin

File: packages/apps-config/src/ui/general/index.ts (hunk 2 above)
Comment (sxlwar) [reply packages/apps-config/src/ui/general/index.ts]: Backgrounds colors are the same too

File: packages/apps-config/src/api/spec/index.ts (hunk 1 above)
Comment (hackfisher) [reply packages/apps-config/src/api/spec/index.ts]: Really? The same spec? @AurevoirXavier

Comment (hackfisher) [reply packages/apps-config/src/api/spec/index.ts]: https://github.com/darwinia-network/darwinia-common/blob/90add536ed320ec7e17898e695c65ee9d7ce79b0/node/runtime/pangoro/src/lib.rs?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L133

Comment (acgxv) [reply packages/apps-config/src/api/spec/index.ts]: Only types are the same.

**Review by hackfisher — APPROVED** [review]

**Pull request merged**